.github/workflows/get-prettyurl-conf.yml

```diff
@@ -0,0 +1,34 @@
+name: get prettyurl conf
```

Comment:
The bot-authored commits are not random. They only get commited if anything in the new htaccess file changed because the nextcloud package was upgraded